org.eclipse.xtext/src/org/eclipse/xtext/resource/containers/StateBasedContainer.java

```diff
@@ -61,22 +66,22 @@ public boolean apply(IEObjectDescription input) {
 
 	@Override
 	public boolean hasResourceDescription(URI uri) {
-		return state.contains(uri);
+		return getState().contains(uri);
```

Comment:
Now that we do have the final getter for the final field, it would be ok to continue using the field here, but each JVM that I know will manage to inline the getter. So this should be ok.